src/api/notification/policies/isAuthenticated.js

```diff
@@ -0,0 +1,25 @@
+
+module.exports = async (ctx, next) => {
+
+  // Check if user is authenticated
+  if (!ctx.state.user) {
+    console.log(ctx)
+    console.log("Not LOGGED IN")
+    return ctx.unauthorized('You must be logged in to access this resource.');
+  }
+
+  // Check if user is authorized to read notifications
+  const userId = ctx.state.user.id; // Get the ID of the logged-in user
+  const notificationUserId = ctx.request.body.user; // Get the user ID from the request body
```

Comment:
Is user an object or just an integer?

Reply:
Was testing the policy attachment to the api. This would also be edited